sharktank/tests/ops/sharded_test.py

```diff
@@ -134,6 +135,108 @@ def testCov2dShardedOutputChannelsOneGroup(self):
         torch.testing.assert_close(actual_result, expected_result)
 
 
+class ElementwiseTest(unittest.TestCase):
+    def testRhsAndLhsShardedAdd(self):
+        a = torch.rand(4, 5, 6, dtype=torch.float32)
+        b = torch.rand(4, 5, 6, dtype=torch.float32)
+
+        expected_result = a + b
+
+        shard_dim = 2
+        shard_count = 3
+        sharded_a = ops.shard(a, dim=shard_dim, count=shard_count)
+        sharded_b = ops.shard(b, dim=shard_dim, count=shard_count)
+        sharded_result = sharded_a + sharded_b
+        actual_result = ops.shard_like(sharded_result, expected_result)
+
+        torch.testing.assert_close(actual_result, expected_result)
+
+    def testBinaryOperators(self):
+        operators = [torch.add, torch.div, torch.fmin, torch.fmax, torch.sub]
```

Comment:
Prefer using `parameterized` (like this: https://github.com/nod-ai/sharktank/blob/ba89559f1c7e8661af977c7764c1d063acb8b401/sharktank/tests/kernels/mmt_block_scaled_q8_test.py#L24)